filebeat/Makefile

```diff
@@ -51,6 +51,12 @@ collect-docs: python-env
 	@mkdir -p docs/modules
 	@${PYTHON_ENV}/bin/python ${ES_BEATS}/filebeat/scripts/docs_collector.py --beat ${BEAT_NAME}
 
+# Generate imports for prospectors
+.PHONY: imports
+imports: python-env
```

Comment:
This should be added to a dependency in collect.

Reply:
donz